deb-package-builder/extensions/cassandra-cpp-driver/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+source ${DEB_BUILDER_DIR}/extensions/functions.sh
+
+echo "Building cassandra cpp driver"
+
+echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
```

Comment:
I probably should just download the files from the backports repo rather than adding a source which could provide other packages that won't be available at runtime.